pkg/types/openstack/defaults/postvalidation/machinepool.go

```diff
@@ -0,0 +1,8 @@
+package defaults
+
+// Defaults that rely on user input to be set should be stored here. This way feedback about incorect user
```

Comment:
What is this file about? Looks like it is not used.

Reply:
accidental merge